docs/src/content/projects/better-dev-screencasts.mdoc

```diff
@@ -0,0 +1,64 @@
+---
+title: Better Dev Screencasts
+type: production
+url: https://betterdevscreencasts.com
+repoUrl: https://github.com/simonswiss/better-dev
+summary: >-
+  A collection of resources to help developers create better screencasts, and
+  level-up their overall video content creation game.
+coverImage:
+  src: >-
+    https://thinkmill-labs.keystatic.net/keystatic-site/images/t1i890vdj0cf/keystatic-site
```

Comment:
This seems to be a keystatic docs screenshot instead of a Better Dev Screencasts 
(I was snooping on your docs site updates and notice this, looking good!)

Reply:
Ooh — good catch! And thank you 🙏🏼 

Updated 👍